src/utils/queryString.ts

```diff
@@ -3,7 +3,7 @@ import map from 'lodash/map'
 
 import { camelCaseProperties, snakeCaseProperties } from './transformObjectProperties'
 
-export type QueryString = Record<string, string | string[] | number | boolean | undefined>
```

Comment:
I know that `string | string[] | number | boolean | undefined` is a poor typing but It still give more information than `any`.

Reply:
After some more research, I've found a better solution :)